docs/src/style/index.md

```diff
@@ -12,6 +12,13 @@ In most circumstances, the standards provided by this style guide should *not* b
 If an exception must me made, the rationale should be included in the respective PR.
 Any `Earthfile` which does not adhere to this style guide will be rejected if no further justification is made.
 
+### Earthly User Defined Commands (UDCs) deprecated starting from Earthly 0.8
```

Comment:
It currently looks like this
![Screenshot 2567-01-16 at 13 01 47](https://github.com/input-output-hk/catalyst-ci/assets/35752733/5b597704-30c0-4175-8a7d-8e84ce7ba5f8)

It should looks like this
![Screenshot 2567-01-16 at 13 04 10](https://github.com/input-output-hk/catalyst-ci/assets/35752733/93e7a479-902a-4ea3-a120-d374effeab08)

Reply:
Wrong information too, UDC not FDC